### PR TITLE
docs(skip): add description and example for the operator

### LINKED ISSUE #4868: [docs]: Suggestions for `skip` documentation improvements
### Documentation Related To Component:

The `skip` operator: 
[skip.ts](https://github.com/ReactiveX/rxjs/blob/b24d3ea85d33b893e4c020fb4fa9f6fa4d05dc0b/src/internal/operators/skip.ts)

### Please check those that apply

- [ ] typo
- [X] documentation doesn't exist
- [ ] documentation needs clarification
- [ ] error(s)/adoption/updates in example
- [X] needs example

### Description Of The Issue

The detailed description is missing. Also, the example as well as seel also links are missing.
We might want to improve this to serve a better more detailed description for the community.

#### Description
The detailed description is missing. Here we should include that:
- it skips values until the sent notifications are equal or less the provided skip count
- it errors if skip count is equal or more than the actual number of emits and source raises an error
- it raises an error if the source raises an error
- it completes if source completes

#### Example
We at least provide a basic example.

#### See Also
We miss the see also section completely.

A suggested list for related operators could look like this:
```typescript
 * @see {@link last}
 * @see {@link skipWhile}
 * @see {@link skipUntil}
 * @see {@link skipLast}
```

In case we also want to reference take operators we also might want to include the following operators:
```typescript
 * @see {@link first}
 * @see {@link take}
 * @see {@link takeWhile}
 * @see {@link takeUntil}
 * @see {@link takeLast}
```

### PR DESCRIPTION
**Pull request by hardikpthv**
**Description:**
- Add description, example, and See Also section for the `skip` operator

**Related issue (if exists):**
- Closes #4868

### CODE REVIEW

File: src/internal/operators/skip.ts [hunk 1]
```diff
@@ -7,6 +7,29 @@ import { OperatorSubscriber } from './OperatorSubscriber';
  * Returns an Observable that skips the first `count` items emitted by the source Observable.
  *
  * ![](skip.png)
+ * 
+ * Skips the values until the sent notifications are equal or less than provided skip count. It raises 
+ * an error if skip count is equal or more than the actual number of emits and source raises an error. An emission
+ * is over when source is reached to its limit.
```
Comment (niklas-wortmann) [inline src/internal/operators/skip.ts:13]: Can you delete this last sentence. I get what you are saying, but I think this causes more confusion than it helps the reader

**Review by niklas-wortmann — CHANGES_REQUESTED** [review]
Hi one minor thing again :D Many thanks for you contributions I really appreciate it

**Comment by hardikpthv** [comment]
> Hi one minor thing again :D Many thanks for you contributions I really appreciate it

You're welcome, @niklas-wortmann! Happy to contribute :)

**Review by niklas-wortmann — APPROVED** [review]
LGTM

**Pull request merged**